core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -198,6 +198,30 @@ public static long parseSpaceSize(String spaceSize) {
   }
 
   /**
+   * Parses a time from millisecond to target unit
```

Comment:
our code style requires a period in the end of the first sentence, or the style checker will fail your PR.